tests/drivers/build_all/sensor/i2c.dtsi

```diff
@@ -549,6 +549,12 @@ test_i2c_vcnl4040: vcnl4040@57 {
 	int-gpios = <&test_gpio 0 0>;
 };
 
+test_i2c_veml7700: veml7700@10 {
+	compatible = "vishay,veml7700";
+	reg = <0x10>;
+	psm-mode = <0x03>;
+};
```

Comment:
Move to the bottom of the file and increment the address from the previous node by one. This is a build test only and the address does not need to match the datasheet